Devscord.DiscordFramework/Services/EmbedMessagesService.cs

```diff
@@ -22,11 +24,7 @@ public Embed Generate(string title, string description, Dictionary<string, Dicti
             var flatValues = new Dictionary<string, string>();
             foreach (var value in values)
             {
-                var valuesString = string.Empty;
-                foreach (var v in value.Value)
-                {
-                    valuesString += $"{v.Key} {v.Value}\n";
-                }
+                var valuesString = value.Value.Aggregate(String.Empty, (a, b) => (b.Key +" "+ b.Value));
```

Comment:
```suggestion
                var valuesString = value.Value.Aggregate((a, b) => $"{a}\n{b.Key} {b.Value}");
```
wydaje mi się, że tak będzie lepiej, tylko sprawdź czy to działa

Reply:
nie kompiluje się wtedy

Reply:
być może to ja źle rozumiem aggregate 
pewnie problem jest z brakiem String.Empty?
a to twoje działa dobrze?
bo nie rozumiem w jaki sposób to łączy skoro bierze pod uwagę tylko zmienną b

Reply:
moje działa :) i Twoja wersja, po dopisaniu "string.Empty" również

Reply:
ale działa twoje również kiedy jest więcej niż jedna linijka? bo dziwię się że nic się nie dzieje z tą zmienną `a`